### PR TITLE
text-field

### PR DESCRIPTION
**Pull request by betaboon**

### CODE REVIEW

File: components/text-field/src/TextField.svelte [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        type="text"
+        bind:this={inputElement}
+        bind:value
+        {id}
+        placeholder={placeholder || ' '}
+      />
+      {#if label}
+        <label for={id}>{label}</label>
+      {/if}
+      <fieldset>
```
[the remark this answers is not in the document]
Comment (betaboon) [reply components/text-field/src/TextField.svelte:61]: `input`, `label` etc are not within the `fieldset` in order for `input:placeholder-shown ~ fieldset` to work.

[the remark this answers is not in the document]
Comment (Florian-Schoenherr) [reply components/text-field/src/TextField.svelte:61]: but just leaving out fieldset and replacing it with a div or whatever won't work?

[the remark this answers is not in the document]
Comment (betaboon) [reply components/text-field/src/TextField.svelte:61]: the you would have to somehow solve the nodge (the left-out border behind the label) which fieldset solves itself

[the remark this answers is not in the document]
Comment (Florian-Schoenherr) [reply components/text-field/src/TextField.svelte:61]: others do seem to do it like this, too (material-ui for example)
image
then I think we should also put aria-hidden on it.

**Pull request merged**